packages/vue-app/template/utils.js

```diff
@@ -260,7 +260,7 @@ export function promisify (fn, context) {
 
 // Imported from vue-router
 export function getLocation (base, mode) {
-  let path = decodeURI(window.location.pathname)
+  let path = decodeURI(window.location.pathname + '/').replace('//', '/') // to get matched with sanitized router.base
```

Comment:
Some important note:

- Having double slash inside path in url is valid and if we do replace, may introduce regressions. What if we just ensure pathname is ending with slash and if not appending?

Some  general notes:

1. (not related to this PR) but can we move this after `if(mode ==='hash')`. It can prevent an extra computed var when we don't need it
2. Can you please put comment above and start first chat with upper case? (`// To get`)

Reply:
You are right, I didn't think about this case and will fix it

Reply:
Fixed